homeassistant/config_entries.py

```diff
@@ -438,7 +466,13 @@ def async_entries(self, domain: Optional[str] = None) -> List[ConfigEntry]:
                 # New in 0.79
                 connection_class=entry.get('connection_class',
                                            CONN_CLASS_UNKNOWN))
-            for entry in config['entries']]
+
+            migrated = await entry.async_migrate(self.hass)
+            any_migrated = True if migrated else migrated
```

Comment:
```py
any_migrated = True if migrated else any_migrated
```

Reply:
Thanks... duh... was kinda tired when I did this :)